File: CSX/Functions and Execution Context/getRemainder.js
function getRemainder(num1, num2) {
  if (num1 > num2) {
    return num1 % num2;
  } else if (num2 > num1) {
    return num2 % num1;
  }
}

console.log(getRemainder(17, 5));
console.log(getRemainder(20, 5));
console.log(getRemainder(8, 22));
console.log(getRemainder(7, 42));

File: CSX - Python/Functions and Execution Context/get_remainder.py
def get_remainder(num1, num2):
    return max(num1, num2) % min(num1, num2)

print(get_remainder(17, 5))
print(get_remainder(20, 5))
print(get_remainder(8, 22))
print(get_remainder(7, 42))
